### DISCUSSION TITLE
How to change existing Rasa->Chatwoot bot token with Docker implemented?

### DISCUSSION THREAD

**Discussion by sohomsaha1** [chatwoot/chatwoot#3793] [Chatwoot APIs] [answered]
I have an existing chatbot setup using Rasa->PHP->Chatwoot setup, I created a new chatwoot account and want to change the bot token, but I do not know how. I am using Docker, and docker-compose. When I use docker-compose up, I have my postgres, rasa, action server, and duckling running. I do not understand how to call a command to generate a new token from my new chatwoot account. As a result, the chatbot does not reply on the live site. Currently, in my chatwoot bot token is for the old account and in my PHP script.

**Comment by pranavrajs** [comment, marked as answer]
Hey Soham, You can generate a new account agent bot using this API (https://www.chatwoot.com/developers/api/#tag/Account-AgentBots). It would return a new agent bot token that can be used to call the API.

Let us know how it goes.
